Libraries/zephyr/MAX/Source/MAX78000/CMakeLists.txt

```diff
@@ -0,0 +1,187 @@
+    ${MSDK_PERIPH_SRC_DIR}/LP
+    ${MSDK_PERIPH_SRC_DIR}/LPCMP
+    ${MSDK_PERIPH_SRC_DIR}/OWM
+    ${MSDK_PERIPH_SRC_DIR}/PT
+    ${MSDK_PERIPH_SRC_DIR}/RTC
+    ${MSDK_PERIPH_SRC_DIR}/SEMA
+    ${MSDK_PERIPH_SRC_DIR}/SPI
+    ${MSDK_PERIPH_SRC_DIR}/SYS
+    ${MSDK_PERIPH_SRC_DIR}/TMR
+    ${MSDK_PERIPH_SRC_DIR}/TRNG
+    ${MSDK_PERIPH_SRC_DIR}/UART
+    ${MSDK_PERIPH_SRC_DIR}/WDT
+    ${MSDK_PERIPH_SRC_DIR}/WUT
+)
+
+if(CONFIG_ARM)
+  zephyr_library_sources(${MSDK_CMSIS_DIR}/Source/system_max78000.c)
+  zephyr_library_sources(${MSDK_PERIPH_SRC_DIR}/SYS/mxc_lock.c)
+elseif(CONFIG_RISCV)
+  zephyr_library_sources(${MSDK_CMSIS_DIR}/Source/system_riscv_max78000.c)
+endif()
+
+zephyr_library_sources(
+    ./max78xxx_system.c
+
+    ${MSDK_PERIPH_SRC_DIR}/CAMERAIF/cameraif_ai85.c
+    ${MSDK_PERIPH_SRC_DIR}/CAMERAIF/cameraif_reva.c
+
+    ${MSDK_PERIPH_SRC_DIR}/ICC/icc_me17.c
+    ${MSDK_PERIPH_SRC_DIR}/ICC/icc_reva.c
+
+    ${MSDK_PERIPH_SRC_DIR}/LP/lp_ai85.c
+
+    ${MSDK_PERIPH_SRC_DIR}/LPCMP/lpcmp_ai85.c
+    ${MSDK_PERIPH_SRC_DIR}/LPCMP/lpcmp_reva.c
+
+    ${MSDK_PERIPH_SRC_DIR}/SYS/mxc_assert.c
+    ${MSDK_PERIPH_SRC_DIR}/SYS/mxc_delay.c
+    ${MSDK_PERIPH_SRC_DIR}/SYS/pins_ai85.c
+    ${MSDK_PERIPH_SRC_DIR}/SYS/sys_ai85.c
+
+    ${MSDK_PERIPH_SRC_DIR}/DMA/dma_me17.c
+    ${MSDK_PERIPH_SRC_DIR}/DMA/dma_reva.c
+
+    ${MSDK_PERIPH_SRC_DIR}/SEMA/sema_me17.c
+    ${MSDK_PERIPH_SRC_DIR}/SEMA/sema_reva.c
+
+    ${MSDK_PERIPH_SRC_DIR}/PT/pt_ai85.c
+    ${MSDK_PERIPH_SRC_DIR}/PT/pt_reva.c
+
+    ${MSDK_PERIPH_SRC_DIR}/WUT/wut_ai85.c
+    ${MSDK_PERIPH_SRC_DIR}/WUT/wut_reva.c
+)
+
+if (CONFIG_ADC_MAX32)
+zephyr_library_sources(
+    ${MSDK_PERIPH_SRC_DIR}/ADC/adc_me17.c
+    ${MSDK_PERIPH_SRC_DIR}/ADC/adc_reva.c
+)
+endif()
+
+if (CONFIG_UART_MAX32)
+zephyr_library_sources(
+    ${MSDK_PERIPH_SRC_DIR}/UART/uart_common.c
+    ${MSDK_PERIPH_SRC_DIR}/UART/uart_ai85.c
+    ${MSDK_PERIPH_SRC_DIR}/UART/uart_revb.c
+)
+endif()
+
+if (CONFIG_GPIO_MAX32)
+zephyr_library_sources(
+    ${MSDK_PERIPH_SRC_DIR}/GPIO/gpio_common.c
+    ${MSDK_PERIPH_SRC_DIR}/GPIO/gpio_ai85.c
+    ${MSDK_PERIPH_SRC_DIR}/GPIO/gpio_reva.c
+)
+endif()
+
+if (CONFIG_SPI_MAX32)
+zephyr_library_sources(
+    ${MSDK_PERIPH_SRC_DIR}/SPI/spi_me17.c
+    ${MSDK_PERIPH_SRC_DIR}/SPI/spi_reva1.c
+)
+endif()
+
+if (CONFIG_I2C_MAX32)
+zephyr_library_sources(
+    ${MSDK_PERIPH_SRC_DIR}/I2C/i2c_me17.c
+    ${MSDK_PERIPH_SRC_DIR}/I2C/i2c_reva.c
+)
+endif()
+
+if (CONFIG_I2S_MAX32)
+zephyr_library_sources(
+    ${MSDK_PERIPH_SRC_DIR}/I2S/i2s_ai85.c
+    ${MSDK_PERIPH_SRC_DIR}/I2S/i2s_reva.c
+)
+endif()
+
+if (CONFIG_WDT_MAX32)
+zephyr_library_sources(
+    ${MSDK_PERIPH_SRC_DIR}/WDT/wdt_common.c
+    ${MSDK_PERIPH_SRC_DIR}/WDT/wdt_me17.c
+    ${MSDK_PERIPH_SRC_DIR}/WDT/wdt_revb.c
+)
+endif()
+
+if (CONFIG_RTC_MAX32 OR CONFIG_COUNTER_RTC_MAX32)
+zephyr_library_sources(
+    ${MSDK_PERIPH_SRC_DIR}/RTC/rtc_me17.c
+    ${MSDK_PERIPH_SRC_DIR}/RTC/rtc_reva.c
+)
+endif()
+
+if (CONFIG_SOC_FLASH_MAX32)
```

Comment:
Please add "CONFIG_HWINFO_MAX32" to if statement of HWINFO driver's dependencies (AES, FLC and TRNG). ([Related PR](https://github.com/analogdevicesinc/msdk/pull/1292))